.env.compose

```
@@ -92,18 +92,23 @@ REDIS_TLS=false
 
-EXPRESS_SESSION_SECRET=gauzy
+# ============================================================================
+# SECURITY: Authentication & session secrets
+# Set each of these to a strong, UNIQUE, random value before deploying, e.g.:
+#     openssl rand -hex 64
+# The API refuses to start in production (NODE_ENV=production and DEMO != true)
+# while any of these is empty or left at a well-known default value, because
+# shared/default secrets let anyone forge authentication tokens and sessions.
+# ============================================================================
+JWT_SECRET=
+EXPRESS_SESSION_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPRESS_SESSION_SECRET key should go before the JWT_SECRET key

(UnorderedKey)